main.go

```diff
@@ -74,6 +74,11 @@ func main() {
 					Name:  "aes-key",
 					Usage: "Encrypt sensitive data in database (length: 16, 24 or 32)",
 				},
+				cli.StringFlag{
+					Name: "logs-location",
+					Value: "/var/log/sshportal",
```

Comment:
Default log location should be `./log` (running `sshportal` should work for most users, even non-admin ones)

Reply:
That makes sense to me. I was sysadmin not that long ago...